hack/common.sh

```diff
@@ -99,6 +90,10 @@ run_WMCO() {
 cleanup_WMCO() {
   local OSDK=$1
   # Remove the operator from openshift-windows-machine-config-operator namespace
+
+  # Revert changes made to the csv
+  sed -i "s|$OPERATOR_IMAGE|REPLACE_IMAGE|g" deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml
```

Comment:
Instead of replacing every time we error out(other comments in this review), you can just catch the error of cleanup command and do the replacement after that....That would cover all the cases...

Reply:
I don't follow how we would be replacing every time we error out. If we error out the first time, how will we reach the second spot where we error out?

Reply:
In my previous comments, i had asked for reverting the `replace_image` change that we are making in run stage...  
this comment says that, instead of doing that, you can just catch `cleanup` error here and print it and do the `sed` afterwards ... 
that would cover all error out conditions